Sources/Fullscreen/PopupConsentView/PopupViewModel.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import Foundation
+
+public protocol PopupViewModel {
```

Comment:
I vote for keeping the model. It will keep the implementation point leaner and it also matches how other views are set up in FinniversKit. However, I'm open to changing this, but then we should change all models, not just this one.

Reply:
What do you think @3lvis ?